userspace/otpilot/src/main.rs

```diff
@@ -148,9 +154,19 @@ fn run() -> TockResult<()> {
 
     //////////////////////////////////////////////////////////////////////////////
 
+    // We assume that we've already done all boot-time checks at this point.
```

Comment:
Is there any way to assert this?

Reply:
This is more of a "design it in such a way that ..." type of comment.